linera-service/src/linera/client_context.rs

```diff
@@ -505,9 +504,9 @@ where
         }
 
         for chain_id in key_pairs.keys() {
-            let mut child_client = self.make_chain_client(*chain_id);
+            let child_client = self.make_chain_client(*chain_id);
```

Comment:
(not strictly this PR) `child_client` suggests that we should probably have written `client` instead of `chain_client` earlier in the function, otherwise it would be `child_chain_client`

Reply:
I agree, but a lot of this will change in a near-future PR when I remove the `ChainClients` type.